__docs__/wonder-blocks-announcer/announcer.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import packageConfig from "../../packages/wonder-blocks-announcer/package.json";
+
+const AnnouncerExample = ({
+    message = "Clicked!",
+    level,
+    debounceThreshold,
+}: AnnounceMessageProps) => {
+    return (
+        <Button
+            onClick={async () => {
+                const idRef = await announceMessage({
+                    message,
+                    level,
+                    debounceThreshold,
+                });
+                /* eslint-disable-next-line */
+                console.log(idRef);
+            }}
+        >
+            Save
+        </Button>
+    );
+};
+type StoryComponentType = StoryObj<typeof AnnouncerExample>;
+
+/**
+ * Announcer exposes an API for screen reader messages using ARIA Live Regions.
+ * It can be used to notify Assistive Technology users without moving focus. Use
+ * cases include combobox filtering, toast notifications, client-side routing,
+ * and more.
+ *
+ * Calling the `announceMessage` function automatically appends the appropriate live regions
+ * to the document body. It sends messages at a default `polite` level, with the
+ * ability to override to `assertive` by passing a `level` argument. You can also
+ * pass a `debounceThreshold` to wait a specific duration before making another announcement.
+ *
+ * To test this API, turn on VoiceOver for Mac/iOS or NVDA on Windows and click the example button.
+ *
+ * ### Usage
+ * ```jsx
+ * import { appendMessage } from "@khanacademy/wonder-blocks-announcer";
+ *
+ * <div>
+ *  <button onClick={() => appendMessage({message: 'Saved your work for you.'})}>
+ *      Save
+ *  </button>
+ * </div>
+ * ```
+ */
+export default {
+    title: "Packages / Announcer",
+    component: AnnouncerExample,
+    decorators: [
+        (Story): React.ReactElement<React.ComponentProps<typeof View>> => (
+            <View style={styles.example}>
+                <Story />
+            </View>
+        ),
+    ],
+    parameters: {
+        componentSubtitle: (
+            <ComponentInfo
+                name={packageConfig.name}
+                version={packageConfig.version}
+            />
+        ),
+        docs: {
+            source: {
+                // See https://github.com/storybookjs/storybook/issues/12596
+                excludeDecorators: true,
+            },
+        },
+    },
+    argTypes: {
+        level: {
+            control: "radio",
+            options: ["polite", "assertive"],
+            defaultValue: "polite",
+        },
+        debounceThreshold: {
+            control: "number",
+            type: "number",
+            description: "(milliseconds)",
+        },
+    },
+} as Meta<typeof AnnouncerExample>;
+
+/**
+ * This is an example of a live region with all the options set to their default
+ * values and the `message` argument set to some example text.
+ */
+export const SendMessage: StoryComponentType = {
+    args: {
+        message: "Here is some example text.",
```

Comment:
So I played around with adding Date.now() or Math.random() to the default message, and it caches it so nothing changes with repeated clicks. The message can be changed manually for this purpose. Any ideas on how to change it dynamically in the way you're imagining?

Reply:
I think to have dynamic messages each time the button is clicked, we would need to update it in the `AnnouncerExample` component. When we configure the story args here, it will only initialize the storybook control values for the first render

For example, if we add `Date.now()` to the message when we call `announceMessage` in the `AnnouncerExample` component, it should be a different message each time!